### PR TITLE
feat(api-client): Check SSO code endpoint

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: packages/api-client/src/main/auth/AuthAPI.ts [hunk 1]
```diff
@@ -109,6 +111,15 @@ class AuthAPI {
 
     return this.client.sendJSON(config).then((response: AxiosResponse) => response.data);
   }
+
+  public async headInitiateLogin(ssoCode: string): Promise<void> {
+    const config: AxiosRequestConfig = {
+      method: 'head',
+      url: `${AuthAPI.URL.INITIATE_LOGIN}/${AuthAPI.URL.INITIATE_LOGIN}/${ssoCode}`,
```
Comment (ffflorian) [inline packages/api-client/src/main/auth/AuthAPI.ts]: `initiate-login/initiate-login`?

File: packages/api-client/src/main/auth/AuthAPI.ts [hunk 2]
```diff
@@ -109,6 +111,15 @@ class AuthAPI {
...
+    };
+
+    await this.client.sendJSON(config);
```
Comment (ffflorian) [inline packages/api-client/src/main/auth/AuthAPI.ts]: The server doesn't return any data?

File: packages/api-client/src/main/auth/AuthAPI.ts [hunk 3]
```diff
@@ -109,6 +111,15 @@ class AuthAPI {
 
     return this.client.sendJSON(config).then((response: AxiosResponse) => response.data);
   }
+
+  public async headInitiateLogin(ssoCode: string): Promise<void> {
+    const config: AxiosRequestConfig = {
+      method: 'head',
```
Comment (atomrc) [inline packages/api-client/src/main/auth/AuthAPI.ts]: Usually, we use uppercase for HTTP method names

Comment (Yserz) [reply packages/api-client/src/main/auth/AuthAPI.ts]: Doch :P

**Pull request merged**